test/testkitUtils.ts

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
+import {mount, ReactWrapper} from 'enzyme';
+import { enzymeTestkitFactoryCreator } from 'wix-ui-test-utils/enzyme';
+import { BaseDriver, DriverFactory } from 'wix-ui-test-utils/driver-factory';
+
+/* TODO: Add this to wix-ui-test-utils or 
+* change wix-ui-test-utils/enzyme's enzymeTestkitFactoryCreator to return the
+* wrapper and wrapperInstance as well.
+*/
+export function createEnzymeDriverFactory<C extends React.Component<any, any>, T extends BaseDriver>(driverFactory: DriverFactory<T>) {
```

Comment:
It is not something we can do in a second, we would have to rethink it. 
For now, I don't think this should block this pr, but it most definitely deserves a card because leaving it here in the backoffice is not a good idea.

Reply:
created this card:
https://wix0864.leankit.com/card/672905743